app/templates/components/sl-split-grid.hbs

```diff
@@ -0,0 +1,111 @@
+<header class="grid-header">
+    <div class="row">
+        <div class="col-sm-6">
+            {{#if view.headerTemplate}}
+                <h3 class="title">
+                    {{partial view.headerTemplate}}
+                </h3>
+            {{/if}}
+        </div>
+
+        {{#if filterName}}
+            <div class="col-sm-6 text-right">
+                {{#sl-button action="toggleFilterPane" size="small"}}{{filterText}}{{/sl-button}}
```

Comment:
Should aria-expanded="false" be added to this component like it was in the changes made to app/templates/components/sl-drop-button.hbs?